README.md

```diff
@@ -308,8 +308,70 @@ just added.
 ## API
 
 <generated_api_start />
+#### The Jest global environment
```

Comment:
I'm wondering if adding simple flow types for all of these in the documentation is helpful or detrimental.

For example:

```js
require(moduleName: string);
// vs
require(moduleName)
```

What do you think?